pkg/db/postgres/run/tests/find_test.go

```diff
@@ -1126,3 +1126,193 @@ func TestRun_Find(t *testing.T) {
 	}
 
 }
+
+// Test to search using the new fields, "Since" and "Duration", added to RunFindQuery
+func TestRun_Find_Add(t *testing.T) {
```

Comment:
Why is this testcase splitted from others? Timeout?

If so, the name should be more nice and expressive.

For example,

```suggestion
func TestRun_Find_Since_and_Duration(t *testing.T) {
```